python/cudf/cudf/core/column/column.py

```diff
@@ -883,7 +883,12 @@ def astype(self, dtype: Dtype, **kwargs) -> ColumnBase:
 
         dtype = pandas_dtypes_to_cudf_dtypes.get(dtype, dtype)
```

Comment:
```suggestion
        dtype = pandas_dtypes_alias_to_cudf_alias.get(dtype, dtype) if isinstance(dtype, str) else pandas_dtypes_to_cudf_dtypes.get(dtype, dtype)
```

We could simplify handling of pandas dtype aliases like this instead of a try/except below.